roles/confluent.test/molecule/Dockerfile-debian-archive.j2

```diff
@@ -11,3 +19,7 @@ RUN apt-get update && \
       openssl \
       unzip \
       curl
+
+# Hack to fix cacerts on debian hosts, preventing confluenthub installs from working
```

Comment:
what is this hack for ? 🤔

Reply:
some installations, like gnupg were failing. I didn't go into depth but this is the same file we use in CP 7.0.x and above